action.yml

```diff
@@ -54,6 +57,8 @@ runs:
             .addHeading('Outdated packages :broom:')
             .addTable([header, ...rows])
             .write()
+          // Set the summary output
+          core.setOutput('summary', core.summary._buffer)
```

Comment:
Seeing the `summary.ts` of `core` ↓ , `.buffer` looks like private and the `stringify()` method may be a better choice.

https://github.com/actions/toolkit/blob/7f5921cdddc31081d4754a42711d71e7890b0d06/packages/core/src/summary.ts#L335

Reply:
Yep, I agree. The reason it ended up like this is that I first just used the `core.summary`, which resulted in a JSON object in the output like `{"_buffer": "<...>"}`. Using `._buffer` solved the issue so I didn't look any further into it, but `stringify` definitely makes sense to use :)

Reply:
Thanks!